src/components/Menu-new/Menu.tsx

```diff
@@ -0,0 +1,112 @@
+import { ReactElement, useCallback, useState } from 'react';
+import { Icon, View, Text } from '@aws-amplify/ui-react';
+import Link from 'next/link';
+import { directory } from './buildDirectory.mjs';
+import styles from './Menu.module.scss';
+import { useRouter } from 'next/router';
+
+export function Menu(): ReactElement {
+  const rootPage = directory;
+  const platformOverviewPage = rootPage.children[0];
+
+  return (
+    <nav className={styles['menu']}>
+      <View marginBottom="16px">
+        <Text fontSize="16px" fontWeight="700">
+          How Amplify works
+        </Text>
+      </View>
+      <ul>
+        {platformOverviewPage.children &&
+          platformOverviewPage.children.map((child) => {
+            return generateListItem(child, 1);
+          })}
+      </ul>
+    </nav>
+  );
+}
+
+const CATEGORY_LEVEL = 1;
+
+function generateListItem(pageNode, level): ReactElement {
+  const router = useRouter();
+  const [open, setOpen] = useState(false);
+  const toggleDrawer = useCallback(() => {
+    setOpen((prevOpen) => !prevOpen);
+  }, []);
+
+  if (!pageNode) {
+    return <></>;
+  }
+
+  const pathname = `/${router.query.platform}/${pathHelper(pageNode.route)}/`;
+  console.log(pathname);
+  console.log('router', router);
+  const current = router.asPath === pathname;
+  console.log(current);
+
+  return (
+    <li
+      tabIndex={0}
```

Comment:
We shouldn't need this tabindex since inside of the li will be the clickable/tabbable thing.